lib/node_modules/@stdlib/blas/base/dtrsm/lib/base.js

```diff
@@ -0,0 +1,257 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Solves matrix equation `op(A) * X = alpha * B` or `X * op(A) = alpha * B` where `alpha` is a scalar, `X` and `B` are m by n matrices, `A` is a unit, or non-unit, upper or lower triangular matrix and `op( A )` is one of `op( A ) = A` or `op( A ) = A ** T`. The matrix `X` is overwritten on `B`.
+*
+* @private
+* @param {string} order - storage layout of `A` and `B`
+* @param {string} side - specifies whether `op( A )` appears on the left or right of `X`
+* @param {string} uplo - specifies whether the upper or lower triangular part of the matrix `A` is supplied
+* @param {string} transa - specifies the form of `op( A )` to be used in matrix multiplication
+* @param {string} diag - specifies whether or not `A` is unit triangular
+* @param {NonNegativeInteger} m - number of rows in `B`
+* @param {NonNegativeInteger} n - number of columns in `B`
+* @param {number} alpha - scalar constant
+* @param {Float64Array} A - input matrix `A`
+* @param {NonNegativeInteger} LDA - stride of the first dimension of `A` (a.k.a., leading dimension of the matrix `A`)
+* @param {NonNegativeInteger} offsetA - starting `A` index
+* @param {Float64Array} B - input matrix `B`
+* @param {NonNegativeInteger} LDB - stride of the first dimension of `B` (a.k.a., leading dimension of the matrix `B`)
+* @param {NonNegativeInteger} offsetB - starting `B` index
+* @returns {Float64Array} `B`
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* var A = new Float64Array( [ 1.0, 3.0, 0.0, 4.0 ] );
+* var B = new Float64Array( [ 5.0, 7.0, 0.0, 8.0 ] );
+*
+* dtrsm( 'row-major', 'left', 'upper', 'none', 'non-unit', 2, 2, 6.0, A, 2, 0, B, 2, 0 );
+* // B => <Float64Array>[ 30.0, 6.0, 0.0, 12.0 ]
+*/
+function dtrsm( order, side, uplo, transa, diag, m, n, alpha, A, LDA, offsetA, B, LDB, offsetB ) { // eslint-disable-line max-len, max-params
```

Comment:
52:1  warning  Function 'dtrsm' has too many statements (122). Maximum allowed is 100  max-statements

Reply:
You can disable the lint rule by adding `max-statements` to the list of already disabled lint rules.